### PR TITLE
Add next js and brah brah

### PR DESCRIPTION
**Pull request by Billy19191**
dfgoifdsjosfurdgoijhfsdognlsdmnglklgnfkdlsnglkjfsklgndlfskgnkldsfnglkfngkldfnglkdf

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --background: #0a0a0a;
+    --foreground: #ededed;
+  }
+}
+
+body {
```
Comment (tong788) [inline src/app/globals.css:17]: Why use this one?

File: src/app/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import type { Metadata } from "next";
+import localFont from "next/font/local";
+import "./globals.css";
+
+const geistSans = localFont({
+  src: "./fonts/GeistVF.woff",
+  variable: "--font-geist-sans",
```
Comment (tong788) [inline src/app/layout.tsx:7]: Isn't mono fonts better?

**Review by tong788 — COMMENTED** [review]
I check font and brah brah

File: src/app/globals.css (hunk 1 above)
Comment (Billy19191) [reply src/app/globals.css:17]: Because we need this to ....., so it's reasonable to use this

File: src/app/layout.tsx (hunk 2 above)
Comment (Billy19191) [reply src/app/layout.tsx:7]: I agree

Comment (Billy19191) [reply src/app/layout.tsx]: I have change it

**Review by tong788 — APPROVED** [review]
Good

**Pull request merged**